src/uri/params/to_www_form.cr

```diff
@@ -0,0 +1,13 @@
+class Object
+  # :nodoc:
+  def to_www_form(builder : URI::Params::Builder, name : String) : Nil
+    builder.add name, to_s
+  end
+end
```

Comment:
**question:** I'm not sure it's a good idea to implicitly implement `#to_www_form` for every type based on `#to_s`.
Maybe it's better to have it opt-in for types who support a meaningful format?